frontend/src/graphql/queries.ts

```diff
@@ -15,3 +15,26 @@ const findAllUserProfiles = async () => {
   return data.users;
 };
 export default findAllUserProfiles;
+
+const joinTownMutation = gql`
+  mutation joinTown($TownJoinRequest: TownJoinRequest!)
+  @rest(type: "Town", path: "/sessions", method: "POST") {
+    isOK
+    response
+    message
+  }
+  `;
+
+const TownJoinRequest = gql`
+  type TownJoinRequest {
+    userName: String!
+    coveyTownID: String!
+  }
+`;
+
+export async function joinTown(TownJoinRequest: any) {
```

Comment:
joinTown is an api that would be called from appropriate function (For eg , Video.ts - they have called this.townsServiceClient.joinTown but now it should be replaced to call this function). So the input parameter that this function receives is a payload of type TownJoinRequest.